### PR TITLE
toggle habitable zones now targets specific bodies

### PR DESCRIPTION
**Pull request by mkinsey**
**Motivation**
Based on feedback, toggle habitable zones should target a specific zone, rather than all present in simulation.

**Test**

This relies on pull request 91 in orbitable/engine. That should be merged and tagged first.

Create two habitable zones. try toggling each in turn and confirm the action is independent.

### CODE REVIEW

File: src/bridge/controllers/BodyController.js [hunk 1]
```diff
@@ -78,11 +78,12 @@ angular.module('bridge.controllers')
       $('#note-sidebar').hide();
     };
 
-    this.toggleHabitable = function(){
-      if($('#habitable-zones').attr("visibility") != "hidden"){
-        $('#habitable-zones').attr("visibility", "hidden");
-      } else {
-        $('#habitable-zones').attr("visibility", "visible");
+    this.toggleHabitable = function(b){
+      console.log("current: ", b.hideHabitable);
```
Comment (benjic) [inline src/bridge/controllers/BodyController.js]: Can you remove this if it was just for debugging.

File: src/bridge/directives/simulation/bodies.js [hunk 2]
```diff
@@ -55,9 +55,13 @@ var BodiesDirective = function(eventPump, simulator, Scale, User) {
...
             }
           }
-        });  
+        });
 
       function update(data) {
+<<<<<<< HEAD
+
+=======
+>>>>>>> master
```
Comment (benjic) [inline src/bridge/directives/simulation/bodies.js]: Whoaoa! You may want to finish merging. :crocodile:

File: src/bridge/directives/simulation/bodies.js [hunk 3]
```diff
@@ -77,20 +81,21 @@ var BodiesDirective = function(eventPump, simulator, Scale, User) {
...
         }
-        
+
         function drawBodies(bodies) {
           bodies
             .attr('cx', (d) => scope.xScale(d.position.x))
             .attr('cy', (d) => scope.yScale(d.position.y))
             .attr('r',  (d) => scope.rScale(d.radius))
             .attr('fill', (d) => d.color)
+            .attr('hideHabitable', (d) => (d.hideHabitable ? d.hideHabitable : false))  // add false if undef
```
Comment (benjic) [inline src/bridge/directives/simulation/bodies.js]: I'm not sure what the `hideHabitable` dom attribute does. Is this supposed to be a class?

**Comment by mkinsey** [comment]
Well that's what I get for committing in a hurry, it should be good to go now.

**Pull request merged**